common/prefix.py

```diff
@@ -8,7 +8,7 @@
 
 class OpenpilotPrefix:
   def __init__(self, prefix: Optional[str] = None, clean_dirs_on_exit: bool = True):
-    self.prefix = prefix if prefix else str(uuid.uuid4())
+    self.prefix = prefix if prefix else str(uuid.uuid4().hex[0:15])
```

Comment:
the C++ implementation uses 15 alphanumeric characters, and I think that looks better than the uuid format with dashes
'550e8400-e29b-41d4-a716-446655440000'